HACKING.rst.txt

```diff
@@ -194,289 +219,6 @@ project and SciPy the community, and it can always use a new pair of hands.
-
-  - The `how to document`_ guidelines
-  - NumPy/SciPy `testing guidelines`_
-  - `SciPy API`_
-  - The `SciPy Roadmap`_
-  - NumPy/SciPy `git workflow`_
-  - How to submit a good `bug report`_
-
-
-FAQ
----
-
-*I based my code on existing Matlab/R/... code I found online, is this OK?*
-
-It depends.  SciPy is distributed under a BSD license, so if the code that you
-based your code on is also BSD licensed or has a BSD-compatible license (e.g.
-MIT, PSF) then it's OK.  Code which is GPL or Apache licensed, has no
-clear license, requires citation or is free for academic use only can't be
-included in SciPy.  Therefore if you copied existing code with such a license
-or made a direct translation to Python of it, your code can't be included.
-If you're unsure, please ask on the scipy-dev mailing list.
-
-*Why is SciPy under the BSD license and not, say, the GPL?*
-
-Like Python, SciPy uses a "permissive" open source license, which allows
-proprietary re-use. While this allows companies to use and modify the software
-without giving anything back, it is felt that the larger user base results in
-more contributions overall, and companies often publish their modifications
-anyway, without being required to.  See John Hunter's `BSD pitch`_.
-
-
-*How do I set up a development version of SciPy in parallel to a released
-version that I use to do my job/research?*
-
-One simple way to achieve this is to install the released version in
-site-packages, by using a binary installer or pip for example, and set
-up the development version in a virtualenv.  First install
-`virtualenv`_ (optionally use `virtualenvwrapper`_), then create your
-virtualenv (named scipy-dev here) with::
-
-    $ virtualenv scipy-dev
-
-Now, whenever you want to switch to the virtual environment, you can use the
-command ``source scipy-dev/bin/activate``, and ``deactivate`` to exit from the
-virtual environment and back to your previous shell.  With scipy-dev
-activated, install first Scipy's dependencies::
-
-    $ pip install NumPy pytest Cython
-
-After that, you can install a development version of Scipy, for example via::
-
-    $ python setup.py install
-
-The installation goes to the virtual environment.
-
-
-*How do I set up an in-place build for development*
-
-For development, you can set up an in-place build so that changes made to
-``.py`` files have effect without rebuild. First, run::
-
-    $ python setup.py build_ext -i
-
-Then you need to point your PYTHONPATH environment variable to this directory.
-Some IDEs (`Spyder`_ for example) have utilities to manage PYTHONPATH.  On Linux
-and OSX, you can run the command::
-
-    $ export PYTHONPATH=$PWD
-
-and on Windows
-
-    $ set PYTHONPATH=/path/to/scipy
-
-Now editing a Python source file in SciPy allows you to immediately
-test and use your changes (in ``.py`` files), by simply restarting the
-interpreter.
-
-
-*Are there any video examples for installing from source, setting up a
-development environment, etc...?*
```

Comment:
Removed question. Link is in new [macOS Quickstart guide](https://github.com/scipy/scipy/pull/10286/files#diff-23d530dd47e20cba4296e7e42f36ac9c).